libsoftwarecontainer/src/gateway/iptableentry.h

```diff
@@ -67,6 +67,18 @@ class IPTableEntry
      */
     ReturnCode applyRules();
 
+    /**
+     * @brief Interprets a rule to iptables applicable string
+     * @return string indicating interpreted rule
+     */
+    std::string interpretRule(Rule rule);
+
+    /**
+     * @brief Interprets defaultTarget rule to iptables applicable string
+     * @return string indicating interpreted policy
```

Comment:
If it's a public member of the class then I think it's good to make the assumption that the user of this class might do erroneous things.